src/NMEA/EngineState.hpp

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include "Validity.hpp"
+#include "time/Stamp.hpp"
+#include "Atmosphere/Temperature.hpp"
+
+/** Information about the engine sensor box values,
+ *  temperatures and revolutions per second.
+ *  Update rate of the engine sensor box is 1 Hz.
+*/
+struct EngineState
+{
+  /** Just one of the 8 bit is used as Flag.
+   *  Lowest bit 0b00000001 = 0x1 set to 1, indicates a valid RPM value.
+   *  All other bits are reseved for further use.
+  */
+  Validity sensbox_flags_available;
+  uint8_t sensbox_flags;
+
+  // The engine box measured revolutions per second.
```

Comment:
API documentation should be doxygen-style. And the meaning should be clarified - what rotation is being measured? The engine's rotation? Before or after the gearbox/strap?